contracts/bancorx/BancorX.sol

```diff
@@ -34,395 +34,577 @@ contract BancorX is IBancorX, TokenHolder, ContractRegistryClient {
         bool completed;
     }
 
-    uint16 public constant version = 4;
+    uint16 public constant VERSION = 4;
```

Comment:
API breaking change

Reply:
Plus, we seem to be using `version` everywhere else

Reply:
It's `BancorX`, so it's not going to be problem (nor we're going to deploy it any time soon).

Reply:
Yeah, we shouldn't have used `version` in `ConverterUpgrader` either. It's a violation of the standard naming conventions, but I've kept it there for backward compatibility, added a linter exception, and a comment.